src/ucp/wireup/address.c

```diff
@@ -1423,6 +1423,39 @@ ucp_address_do_pack(ucp_worker_h worker, ucp_ep_h ep, void *buffer, size_t size,
     return UCS_OK;
 }
 
+ucs_status_t ucp_address_length(ucp_worker_h worker, ucp_ep_h ep,
+                                const ucp_tl_bitmap_t *tl_bitmap,
+                                unsigned pack_flags,
+                                ucp_object_version_t addr_version,
+                                size_t *size_p)
+{
+    ucp_address_packed_device_t *devices;
+    ucp_rsc_index_t num_devices;
+    ucs_status_t status;
+    size_t size;
+
+    if (ep == NULL) {
+        pack_flags &= ~UCP_ADDRESS_PACK_FLAG_EP_ADDR;
+    }
```

Comment:
it should be done outside like other flags

Reply:
Done

Reply:
I'm going to remove the logic while checking the required packed ucp_ep address length because the ```ep``` can't be NULL in client and server connect function. If placing the ```if (ep == NULL)``` in  ```ucp_cm_ep_priv_space_is_enough``` like dealing with checking other flags, it'll hit coverity error.